src/index.ts

```diff
@@ -58,10 +44,33 @@ export default async function watchAndNotify(mongoData: MongoDataType, rabbitDat
     mongoConnection(mongoData, rabbitData, options);
 }
 
+
 function sleep(ms: number) {
     return new Promise(resolve => setTimeout(resolve, ms));
 }
 
+/**
+ * Get rabbit health status
+ * @returns {boolean} - true if healthy
+ */
+export function getRabbitHealthStatus(): boolean{
+    return !menash.isClosed && menash.isReady;
+}
+
+/**
+ * Get mongo connection health status
+ * @returns {boolean} - true if healthy
+ */
+export function getMongoHealthStatus(): boolean {
+    return mongoConn.isConnected();
+}
+
+/**
+ * Start mongo connection
+ * @param {MongoDataType}   mongoData   - mongo uri and collection name
+ * @param {RabbitDataType}  rabbitData  - rabbit data
+ * @param {MTROptions}      options     - mongo to rabbit options
```

Comment:
Why all the curly-braces? It it a known convention?

Reply:
yes